content/blog/2021-02-17-cml-runner-prerelease.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 'Pre-Release Notes: Automatically Train Models in the Cloud with CML'
+date: 2021-02-17
+description: |
+  New features are here to make launching cloud 
+  compute for continuous integration workflows 
+  shorter, sweeter and easier than ever. Plus, 
+  a new GitHub Action to setup CML means more 
+  ways to use CML without our Docker container.
+descriptionLong: |
+  New features are here to make launching cloud 
+  compute for continuous integration workflows 
+  shorter, sweeter and easier than ever. Plus, 
+  a new GitHub Action to setup CML means more 
+  ways to use CML without our Docker container.
+picture: 2021-02-17/gh_actions_art.png
```

Comment:
The only thing I see that's picky, but off, is that in the header image, the 'f' in for is too close to the 'p' in Setup. That second part needs to be spaced down a skosh.